scripts.js

```diff
@@ -70,5 +75,138 @@ function divisible([num]) {
 function cleanNames(input) {
     return input.map(val => val.trim());
  }
- const names = [" Eddah ", " Karambu", "Gikunda"];
- console.log(cleanNames(names));
\ No newline at end of file
+
+console.log(cleanNames([" Eddah ", " Karambu", "Gikunda"]))
+ 
+
+
+
+function divibleByThree(numbers){
+    const len = numbers.length
+    let num = []
+    for(let i = 0; i < len; i++) {
+        if(numbers[i] % 3 === 0){
+           num.push(numbers[i])
+        }
+    }  
+      return num
+    }
+console.log(divibleByThree([2, 3, 4, 5, 6, 7, 8, 9]))
+    
+
+// Write a function that given an array checks whether the array is empty and
+// returns true if empty and false if not empty.
+
+
+function emptyArray(array){
+    const len = emptyArray.length    
+        if(array.length === 0){
+           return false
+        } else {
+            return false
+        }
+    }      
+    
+    console.log(emptyArray([2,3,4,5,6,7,8,9]))
+
+    // Write a function checkIfExists that accepts an array and a string and checks if
+    // that string is in the array. It should return false if the string is not in array
+    // and true if the string is in array e.g
+    // checkIfExists ([‘chair’, ‘door’, ‘table`], ‘door’) return true
+    function checkIfExists(array,str) {
+        if (array.includes(str)) {
+            return true
+        }else{
+            return false
+        }      
+    }   console.log(checkIfExists(['eddah','yui'],'yuiiii'))
+
+
+
+   
+    // 10. Create a function `fill` that takes number of items in the array and a
+    // value returns an array with the same number of items as given e.g
+    // fill(4, ‘b’) returns [‘b’, ‘b’, ‘b’, ‘b’]
+
+       
+    const checkOccurrence = (num, element) => { 
+        let number =[]
+          for(let i =0; i<num; i++){
+            number.push(element)
+          }
+        return number      
+  };
+      console.log(checkOccurrence(4,'eddah'))
+
+// 6th July 2022
+      // 1. Write a JavaScript function to check whether an `input` is an array or not. 
+      // Test Data : 
+      // console.log(is_array('w3resource')); 
+      // console.log(is_array([1, 2, 4, 0])); 
+
+      const array = (input) =>{
+        if (toString.call(input) === "[object Array]")
+          return true;
+        return false;   
+          };
+      console.log(array('w3resource'));
+      console.log(array([1, 2, 4, 0]));
+      
+
+//       2. Write a JavaScript program to sort the items of an array. Sample array : var arr1 = [ 3, 8, 7, 6, 5, -4, 3, 2, 1 ]; 
+// Sample Output : -4,-3,1,2,3,5,6,7,8 
+      
+      const names = ["Karambu", "Dedan", "Muthomi", "Erick", "Eddah"];
+      console.log(names.sort());
+      
+      var priceList = [ 3, 8, 7, 6, 5, -4, 3, 2, 1];
+      priceList.sort();
+      console.log(priceList)
```

Comment:
have this in function and please do not use the sort method. Loop through the arrays